src/Repository/EventRegistrationRepository.php

```diff
@@ -173,26 +173,61 @@ public function countEventParticipantsInReferentManagedArea(Adherent $referent,
     {
         $this->checkReferent($referent);
 
-        $query = $this->createQueryBuilder('event_registrations')
-            ->select('DISTINCT event_registrations.emailAddress, COUNT(DISTINCT event_registrations) AS count, YEAR_MONTH(event.beginAt) as yearmonth')
-            ->join(Event::class, 'event', Join::WITH, 'event_registrations.event = event.id')
-            ->leftJoin(Adherent::class, 'adherent', Join::WITH, 'adherent.uuid = event_registrations.adherentUuid')
+        $query = $this->createQueryBuilder('eventRegistrations')
+            ->select('COUNT(DISTINCT eventRegistrations) AS count, YEAR_MONTH(event.beginAt) as yearmonth')
```

Comment:
We should not modify the count because this method is already used in another endpoint to calcul number of different persons participated in all events in managed area. 
We should add a parameter to define a count calculation or create another method.